test-automation-school/src/main/java/test/automation/school/pageobjects/SearchPage.java

```diff
@@ -0,0 +1,73 @@
+package test.automation.school.pageobjects;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+/**
+ * Created by lina.pugo on 5/17/2017.
+ */
+public class SearchPage {
+    private WebDriver driver;
+    private WebDriverWait wait;
+
+    @FindBy(xpath = "//input[@id='ptxt']")
+    public WebElement findSearchfield;
+
+    @FindBy(xpath = "//input[@value='Искать']")
+    public WebElement searchButton;
+
+    @FindBy(xpath = "//select[@name='search_region']")
```

Comment:
Firstly, if you look at this dropdown in DOM, then you will find that it contains "id" attribute and it doesn`t look like dynamically generated. And, as you probably should know - "The most preferred, the easiest and efficient way to locate an element on a web page is By ID. Id will the attribute on the page which will be unique like you bank account number or employee id. Ids are the safest and fastest locator option and should always be the first choice even when there are multiple choices." Check http://www.seleniumhq.org/docs/03_webdriver.jsp#locating-ui-elements-webelements and http://blog.thedigitalgroup.com/rohits/2015/09/02/locator-strategies-in-selenium-webdriver/